packages/postcss-syntax/src/parse.ts

```diff
@@ -91,6 +95,33 @@ export const parse = (css: string | { toString(): string }, opts?: ParserOptions
   return root;
 };
 
+function parseGriffelStyles(
+  code: string,
+  filename: string,
+  pluginOpts: BabelPluginOptions,
+  silenceParseErrors: boolean,
+) {
+  try {
+    const { metadata } = transformSync(code, {
+      filename,
+      pluginOptions: {
+        ...pluginOpts,
+        generateMetadata: true,
+      },
+    });
+
+    return metadata;
+  } catch (error) {
+    console.warn('Failed to parse Griffel styles');
+    console.warn(error);
+    if (silenceParseErrors) {
+      return null;
```

Comment:
What about console.warn only when silenceParseErrors?

Reply:
so we still have some way of tracing the parse error